packages/calling-component-bindings/src/callControlSelectors.ts

```diff
@@ -145,16 +145,20 @@ export const devicesButtonSelector: DevicesButtonSelector = reselect.createSelec
   [getDeviceManager],
   (deviceManager) => {
     return {
-      microphones: deviceManager.microphones,
-      speakers: deviceManager.speakers,
-      cameras: deviceManager.cameras,
+      microphones: filterBlankNames(deviceManager.microphones),
+      speakers: filterBlankNames(deviceManager.speakers),
+      cameras: filterBlankNames(deviceManager.cameras),
       selectedMicrophone: deviceManager.selectedMicrophone,
       selectedSpeaker: deviceManager.selectedSpeaker,
       selectedCamera: deviceManager.selectedCamera
     };
   }
 );
 
+function filterBlankNames<T extends { name: string }>(devices: T[]): T[] {
```

Comment:
The name `filterBlankNames` doesn't seem right semantically.
Might I suggest something simpler like `removeBlankNameDevices` ?

Reply:
Thanks. Updated.